docs/docs/command-line-interface.mdx

```diff
@@ -58,6 +58,18 @@ LOG_LEVEL_LIBRARIES=DEBUG LOG_LEVEL_MATPLOTLIB=DEBUG rasa shell --verbose
 
 As an aside, CLI log level sets the level at the root logger (which has the important handler - `coloredlogs` handler); this means even if an environment variable sets a library logger to a lower level, the root logger will reject messages from that library. If not specified, the CLI log level is set to `INFO`.
 
+## Custom logging configuration
+
+:::info New in 4.0
```

Comment:
😅 I think it'd be new in `3.4`

Reply:
According to [this Miro board](https://miro.com/app/board/uXjVOjfY7rY=/?fromRedirect=1), the next planned release is a major 🤔

Reply:
I don't have access but I don't think it's the case. I haven't heard anything about it. Can you change to 3.4?